packages/presets/src/ai/messages/text.ts

```diff
@@ -56,51 +52,58 @@ export class AIAnswerTextPreview extends WithDisposable(LitElement) {
   @property({ attribute: false })
   answer!: string;
 
-  private _previewDoc: Doc | null = null;
+  private _doc!: Doc;
+  private _answers: string[] = [];
```

Comment:
It looks like you don't need an array here since you only care about the last item